resources/yaml/mediafusion.yaml

```
@@ -82,13 +78,19 @@ search:
         - name: regexp
           args: "\\b(\\d+(?:\\.\\d+)? [MG]B)\\b"
     seeders:
-      selector: description
-      filters:
-        - name: regexp
-          args: "(?<=👤 )(\\d+)"
+      text: "1"
+    leechers:
+      text: "1"
+    details:
+      text: "{{ .Config.sitelink }}{{ if .Result.category_is_tv_show }}/stream/series/{{ .Query.IMDBID }}:{{ .Query.Season}}:{{ .Query.Ep }}.json{{ else }}/stream/movie/{{ .Query.IMDBID }}.json{{ end }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)